JavaFXUserInterface/src/main/java/Controllers/SystemInfoController.java

```diff
@@ -188,15 +201,19 @@ else if (series.getData().size() != measurements.size()) {
      * @author Jeff
      */
     public void attachEvents() {
-        systemMenuBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, "/Systems"));
-        //Attach link to systemTypeMenuBtn to go to Utilities.SystemTypeList.fxml
+        systemMenuBtn.setOnMouseClicked(mouseEvent -> {
+            timelines.forEach(Timeline::stop);
+            uiUtilities.changeScene(mouseEvent, "/Systems");
+        });
+        //Attach link to systemTypeMenuBtn to go to SystemTypeList.fxml
         systemTypeMenuBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, "/SystemTypeList"));
         deleteBtn.setOnMouseClicked(this::deleteDialog);
```

Comment:
Timelines need to also be stopped here when the page redirects to the Systems page.

Reply:
this is what  timelines.forEach(Timeline::stop); does